src/Routes/ImageManagerDetail/DetailsHeader.js

```diff
@@ -65,90 +65,100 @@ const DetailsHead = ({ imageData, imageVersion, openUpdateWizard }) => {
 
   return (
     <>
-      <Breadcrumb>
-        <BreadcrumbItem>
-          <Link to={paths['manage-images']}>Manage Images</Link>
-        </BreadcrumbItem>
-        {imageVersion ? (
+      {data?.image_set && !data?.image_set?.Account ? (
```

Comment:
how about an helper function to check this?

```js
const isAccountMissing = (imageSet) => imageSet && !imageSet?.Account
```

Reply:
Great suggestion, Thanks @boaz0!